client/galaxy/scripts/mvc/workflow/workflow.js

```diff
@@ -0,0 +1,201 @@
+/** Workflow view */
+define( [ 'mvc/form/form-view',
+          'mvc/ui/ui-misc',
+          'mvc/ui/ui-select',
+          'mvc/tool/tool-form-composite' ], function( Form, Ui, Select, ToolForm ) {
+
+    /** View of the main workflow list page */
+    var View = Backbone.View.extend({
+
+        initialize: function() {
+            this.setElement( '<div/>' );
+            this.render();
+        },
+
+        render: function() {
+            var self = this,
+                min_query_length = 3;
+            $.getJSON( Galaxy.root + 'api/workflows/', function( workflows ) {
+                var $el_workflow = null;
+                // Add workflow header
+                self.$el.empty().append( self._templateHeader() );
+                // Add user actions message if any
+                self.build_messages( self );
+                $el_workflow = self.$el.find( '.user-workflows' );
+                // Add the actions buttons
+                $el_workflow.append( self._templateActionButtons() );
+                if( workflows.length > 0) {
+                    $el_workflow.append( self._templateWorkflowTable( self, workflows) );
+                    self.adjust_actiondropdown( $el_workflow );
+                    // Register delete and run workflow events
+                    _.each( workflows, function( wf ) {
+                        self.confirm_delete( self, wf );
+                    });
+                    // Register search workflow event
+                    self.search_workflow( self, self.$el.find( '.search-wf' ), self.$el.find( '.workflow-search tr' ), min_query_length );
+                }
+                else {
+                    $el_workflow.append( self._templateNoWorkflow() );
+                }
+            });
```

Comment:
No, this is correct, unless I misread your concern.   The following is a single statement, not the opening of a block:
```
var self = this,
      min_query_length = 3;
```

Personally I think it's less clear than explicit var declarations, but multiple assignment like this is fine.